core/base_service_test.go

```diff
@@ -143,6 +197,53 @@ func TestGoodResponseNonJSON(t *testing.T) {
 	assert.Equal(t, expectedResponse, actualResponse)
 }
 
+// Test a text response.
+func TestGoodResponseText(t *testing.T) {
+	expectedResponse := "This is a text response."
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-type", "text/plain")
+		fmt.Fprint(w, expectedResponse)
+	}))
+	defer server.Close()
+
+	builder := NewRequestBuilder("GET")
+	_, err := builder.ConstructHTTPURL(server.URL, nil, nil)
+	assert.Nil(t, err)
+	builder.AddHeader("Content-Type", "Application/json").
```

Comment:
Did you intend to pass "application/json" here?  I suppose it is fine, just might be confusing to someone reading this code later.

Reply:
Yeah those statements were carried forward from the test method I copied from, but it turns out none of the base_service test methods need to set the content type or the version query param so I just remove it from all of the methods.